pyiceberg/io/pyarrow.py

```diff
@@ -889,6 +889,12 @@ def _construct_fragment(fs: FileSystem, data_file: DataFile, file_format_kwargs:
     return _get_file_format(data_file.file_format, **file_format_kwargs).make_fragment(path, fs)
 
 
+def _read_delete_file(fs: FileSystem, data_file: DataFile, schema: "pa.Schema") -> pa.Table:
+    delete_fragment = _construct_fragment(fs, data_file, file_format_kwargs={"pre_buffer": True, "buffer_size": ONE_MEGABYTE})
+    table = ds.Scanner.from_fragment(fragment=delete_fragment, schema=schema).to_table()
+    return table
```

Comment:
nit: this might be a good place to start introducing a `PositionalDelete` class and we can get rid of the custom `_get_positional_file_schema` function